src/app/admin-page/student-form/student-form.component.html

```diff
@@ -1,3 +1,5 @@
+
+<mat-dialog-content>
```

Comment:
Just a suggestion:
```suggestion
<div mat-dialog-content>
```

should work the same or even solve the problem :)